packages/gatsby-theme-docs/package.json

```diff
@@ -41,29 +41,30 @@
     "@videojs/themes": "^1.0.1",
     "assert": "2.0.0",
     "docsearch.js": "2.6.3",
-    "gatsby-core-utils": "3.25.0",
-    "gatsby-plugin-emotion": "7.25.0",
-    "gatsby-plugin-feed": "4.25.0",
-    "gatsby-plugin-google-gtag": "4.25.0",
+    "gatsby-core-utils": "4.6.0",
+    "gatsby-plugin-emotion": "8.6.0",
+    "gatsby-plugin-feed": "5.6.0",
+    "gatsby-plugin-google-gtag": "5.6.0",
     "gatsby-plugin-hubspot": "2.0.0",
-    "gatsby-plugin-image": "2.25.0",
-    "gatsby-plugin-manifest": "4.25.0",
-    "gatsby-plugin-mdx": "3.20.0",
-    "gatsby-plugin-sharp": "4.25.0",
+    "gatsby-plugin-image": "3.6.0",
+    "gatsby-plugin-manifest": "5.6.0",
+    "gatsby-plugin-mdx": "patch:gatsby-plugin-mdx@3.20.0#./patches/gatsby-plugin-mdx.patch",
```

Comment:
Oh, this is cool. It means we don't even have to touch the repos that use the kit :-).   I had only read the docs which were not very specific about how the patch is held and maintained.

Reply:
It's a cool feature indeed 👍